build/build.sh

```diff
@@ -4,19 +4,19 @@ export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
 export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
 export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
 
-export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$http_proxy_port"
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
 
 REMOTE_URL=https://github.com/secure-device-onboard/iot-platform-sdk
 REMOTE_BRANCH=master
 
-if [ "$use_remote" = "1" ]; 
-    then 
+if [ "$use_remote" = "1" ];
+    then
```

Comment:
Keep 'then' at the same indent level of if.

if [ condition ]; then
<indent>statement
else
<indent>statement
fi

or 

if [ condition ]; 
then
<indent>statement
else
<indent>statement
fi

Reply:
Done. Indentation changed.